lib/DUAL_MIT.tpl

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013-2015 StrongLoop, Inc.
+Copyright (c) 2013-2015 StrongLoop, Inc<%= others %>.
```

Comment:
I believe the `.` should be before the expansion, otherwise adding other contributors causes a spelling mistake.